packages/engineer/bin/engineer.js

```diff
@@ -1,17 +1,3 @@
 #!/usr/bin/env node
-const { normalize, dirname } = require('path');
 
-if (__dirname.endsWith(normalize('/packages/engineer/bin'))) {
-    const configFilePath = require.resolve('../../../tsconfig.json');
-    const { createNodeExtension } = require('@ts-tools/node');
-    const nodeExtension = createNodeExtension({ configFilePath });
-    require.extensions['.ts'] = nodeExtension;
-    require.extensions['.tsx'] = nodeExtension;
-
-    const { options: tsconfigPathsOptions } = require('tsconfig-paths/lib/options');
-    tsconfigPathsOptions.cwd = dirname(configFilePath);
-    require('tsconfig-paths/register');
-    require('../src/cli');
-} else {
-    require('../cjs/src/cli');
-}
+require('../dist/cli.js');
```

Comment:
doesn't this mean I have to build for every change I make?

Reply:
yes.